docs/tools/libzbd.md

```diff
@@ -237,8 +244,8 @@ Zone 00014: swr, ofst 00000469762048, len 00000033554432, cap 00000033554432, wp
 Zone 00015: swr, ofst 00000503316480, len 00000033554432, cap 00000033554432, wp 00000503316480, em, non_seq 0, reset 0
 ```
 
-The same zone information can also be obtained in csv format to facilitate
-parsing using scripting languages, including shell scripts.
+The same zone information can also be obtained in csv format so that it can be 
+parsed by scripting languages (including shell scripts).
```

Comment:
"so that it can be more easily parsed by..." would be better, Normal output can be parsed too, but it is harder. CSV format simplifies parsing.

Reply:
In laboring to be brief, I became obscure.

It has been worded the way you suggested.